Specs/Scene/Model/Model3DTileContentSpec.js

```diff
@@ -1006,14 +1006,8 @@ describe(
           pointCloudWithPerPointPropertiesUrl
```

Comment:
the comment above this test should be changed / deleted; a `ModelFeatureTable` is no longer being created